pytest_tests/steps/cluster_test_base.py

```diff
@@ -16,19 +18,18 @@ def fill_mandatory_dependencies(self, cluster: Cluster, client_shell: Shell):
         ClusterTestBase.cluster = cluster
         yield
 
-    @allure.title("Tick {epochs_to_tick} epochs")
-    def tick_epochs(self, epochs_to_tick: int):
-        for _ in range(epochs_to_tick):
-            self.tick_epoch()
-
     def tick_epoch(self):
         epoch.tick_epoch(self.shell, self.cluster)
 
-    def wait_for_epochs_align(self):
-        epoch.wait_for_epochs_align(self.shell, self.cluster)
-
     def get_epoch(self):
         return epoch.get_epoch(self.shell, self.cluster)
 
     def ensure_fresh_epoch(self):
         return epoch.ensure_fresh_epoch(self.shell, self.cluster)
+
+    @allure.step("Tick epochs and wait for epoch alignment")
+    def tick_epochs_and_wait(self, epochs_to_tick: int):
+        current_epoch = self.get_epoch()
+        for _ in range(epochs_to_tick):
+            self.tick_epoch()
+            epoch.wait_for_epochs_align(self.shell, self.cluster, current_epoch)
```

Comment:
is it ok that current_epoch won't be really current? shouldn't it be updated inside the cycle?

Reply:
> is it ok that current_epoch won't be really current? shouldn't it be updated inside the cycle?

That's right - we are comparing the current epoch and the epoch that will be after the ticking.